cli/azd/cmd/infra_create.go

```diff
@@ -234,12 +233,15 @@ func (ica *infraCreateAction) Run(ctx context.Context, cmd *cobra.Command, args
 		printWithStyling(
 			"Provisioning Azure resources can take some time.\n\nYou can view detailed progress in the Azure Portal:\n%s",
 			deploymentURL)
-		//fmt.Fprintf(colorable.NewColorableStdout(), "Provisioning Azure resources can take some time.\n\nYou can view detailed progress in the Azure Portal:\n%s", deploymentURL)
 
-		err = spin.RunWithUpdater("Creating Azure resources ", deployAndReportProgress,
-			func(s *yacspin.Spinner, deploySuccess bool) {
-				s.StopMessage("Created Azure resources\n")
-			})
+		spinner := spin.New("Creating Azure resources")
+		_ = spinner.Start()
+		err = deployAndReportProgress(spinner.Title)
+		_ = spinner.Stop()
```

Comment:
Should we checking for errors here after the calls to `Start` and `Stop` ?

Reply:
@wbreza Do you mean "checking for the error result from `Start()` and `Stop()`"? They are guaranteed to succeed here, since `Start` and `Stop` can only fail if the spinner wasn't already started or stopped which we can verify statically to be the case. That being said...

@ellismg Should we be treating these as closer to dev asserts? 
```go
assertNotErr(error Err, message string) {
  if isDevBuild() && error != nil {
   panic(message)
  } else {
   log.ErrorF(message)
  }
}
```

Reply:
Actually, I might be in more favor of removing `err` from being returned based on our [other discussion](https://github.com/Azure/azure-dev/pull/166#discussion_r928057714)

Reply:
Going to move forward with removing the `error` returned since the assertion vs. the action is a meaningless difference for us.